pkg/volume/configmap/configmap.go

```diff
@@ -194,6 +194,9 @@ func (b *configMapVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if err := wrapped.SetUpAt(dir, fsGroup); err != nil {
 		return err
 	}
+	if err := volumeutil.MakeNestedMountpoints(b.volName, dir, b.pod); err != nil {
```

Comment:
Seems disturbing to use `b` here and `*b` here: https://github.com/kubernetes/kubernetes/pull/60515/commits/25f34665f8127ec3466c06e733da423772a17f48#diff-a3b28c0b2e7de9ad275e0880d0764c8dR187

But it's like the original PR, so perfect :)